prompting/utils/clean_generation.py

```diff
@@ -0,0 +1,73 @@
+import bittensor as bt
+
+
+class GenerationCleaner:
+    def __init__(self) -> None:
+        # TODO: could create these with params?
+        self.cleaning_pipelines = {
+            "question-answering": [
+                self.remove_quotes,
```

Comment:
My feeling is this should live in each task class?

Reply:
definitely. My end goal actually is to have some type of registry, but I guess these can be moved to the task for now

Reply:
Considering that all those cleaning functions are pure, one approach would be isolating them from the class into a dictionary or other type of mapping, which could be used by the task to perform the post processing. Two solutions that I see include:

1. A task define its own post-processing mechanism by key (e.g. `remove_quotes`) and receives a post process pipeline from the outside (similar to `llm_pipeline`)
2. A task define its own post-processing mechanism by key and the task instantiates the pipeline in its creation